src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -5749,6 +5748,50 @@ namespace HeatBalanceSurfaceManager {
 
     void CalcHeatBalanceInsideSurf(ZoneTempPredictorCorrectorData &dataZoneTempPredictorCorrector, Optional_int_const ZoneToResimulate) // if passed in, then only calculate surfaces that have this zone
     {
+
+        if (calcHeatBalInsideSurfFirstTime) {
```

Comment:
Consolidated common initialization blocks here in `CalcHeatBalanceInsideSurf` before splitting off to `CalcHeatBalanceInsideSurf2` or `CalcHeatBalanceInsideSurf2CTFOnly`.